src/components/Tabs.test.tsx

```diff
@@ -78,17 +79,114 @@ describe("TabsWithContent", () => {
   it("hides the tabs if only a single tab is enabled", async () => {
     // Given only the 1st tab is enabled
     const testTabs: Tab<TabValue>[] = [
-      { name: "Tab 1", value: "tab1", render: () => <TestTabContent title="Tab 1 Content" /> },
-      { name: "Tab 2", value: "tab2", disabled: true, render: () => <TestTabContent title="Tab 2 Content" /> },
-      { name: "Tab 3", value: "tab3", disabled: true, render: () => <TestTabContent title="Tab 3 Content" /> },
-      { name: "Tab 4", value: "tab4", disabled: true, render: () => <TestTabContent title="Tab 4 Content" /> },
+      { name: "Tab 1", value: "tab1", render: () => <TestTabContent content="Tab 1 Content" /> },
+      { name: "Tab 2", value: "tab2", disabled: true, render: () => <TestTabContent content="Tab 2 Content" /> },
+      { name: "Tab 3", value: "tab3", disabled: true, render: () => <TestTabContent content="Tab 3 Content" /> },
+      { name: "Tab 4", value: "tab4", disabled: true, render: () => <TestTabContent content="Tab 4 Content" /> },
     ];
-    const r = await render(<TabsWithContent tabs={testTabs} onChange={() => {}} selected="tab1" />);
+    const r = await render(<TabsWithContent tabs={testTabs} onChange={() => {}} selected="tab1" />, withRouter());
     // Then the tabs are not even in the dom
     expect(() => r.tabs_tab1()).toThrow("Unable to find an element");
     // But the selected tab's content is shown
     expect(r.tab_panel().textContent).toBe("Tab 1 Content");
   });
+
+  it("renders tabs as links", async () => {
+    const router = withRouter("/tab1");
+    // Given tabs with `path` values
+    const testTabs: RouteTab<string>[] = [
+      { name: "Tab A", path: "/tab1", href: "/tab1", render: () => <TestTabContent content="Tab 1 Content" /> },
+      { name: "Tab B", path: "/tab2", href: "/tab2", render: () => <TestTabContent content="Tab 2 Content" /> },
+    ];
+    const r = await render(<TabsWithContent tabs={testTabs} />, router);
```

Comment:
What happens when you give `selected` to this component? Would the path or selected take priority?